src/libraries/Microsoft.Bcl.Cryptography/tests/PemEncodingTests.cs

```diff
@@ -0,0 +1,308 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
```

Comment:
Direct copy of the .NET 5 implementation. Only thing that changed was the namespace.